internal/factory/container/container.go

```diff
@@ -27,7 +27,7 @@ import (
 	v1 "github.com/opencontainers/image-spec/specs-go/v1"
 	rspec "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/opencontainers/runtime-tools/generate"
-	"github.com/opencontainers/runtime-tools/validate"
+	validate "github.com/opencontainers/runtime-tools/validate/capabilities"
```

Comment:
this is expected or some artifact from rebases?

Reply:
In runtime-tools we shuffled some functions around a bit a while ago (github.com/xeipuuv/gojson* isolation). Unfortunately in addition to properly bridging to the relocated functions, the original ones were also marked as deprecated. Our cri-o configuration asks golangci-lint to complain about using deprecated functions. So I had to change this to use the new location.